R/cre.R

```diff
@@ -24,8 +25,8 @@
 #' @export
 #'
 cre <- function(y, z, X, ratio_dis, ite_method_dis, ite_method_inf,
-                include_ps_dis = NA, include_ps_inf = NA, ntrees, min_nodes,
-                max_nodes, t, q, rules_method) {
+                include_ps_dis = NA, include_ps_inf = NA, ntrees_rf, ntrees_gbm,
```

Comment:
This can be a temporal solution; however, it is not a good practice. You can either use ... to allow the users to pass any parameter they want or pass all parameters in the list of parameters.  The latter should be easier.